homeassistant/components/powerwall/binary_sensor.py

```diff
@@ -101,7 +101,10 @@ def unique_id(self) -> str:
     @property
     def is_on(self) -> bool:
         """Grid is online."""
-        return self.data.grid_status == GridStatus.CONNECTED
+        return self.data.grid_status in [
```

Comment:
This one could be a set constant as well

Reply:
Ok to include in this PR?

Reply:
Normally I'd ask for another one but its small enough that I'm ok with it.  If we end up with more changes to this platform lets move it.